eos/saveddata/fit.py

```diff
@@ -461,11 +461,16 @@ def calculateModifiedAttributes(self, targetFit=None, withBoosters=False, dirtyS
         timer = Timer(u'Fit: {}, {}'.format(self.ID, self.name), logger)
         logger.debug("Starting fit calculation on: %r, withBoosters: %s", self, withBoosters)
 
+        try:
+            projectionInfo = self.getProjectionInfo(targetFit.ID)
+        except:
+            pass
+        else:
+            logger.debug("ProjectionInfo: %s", projectionInfo)
+
```

Comment:
@Ebag333  Ah, I see what you're talking about now.

In this situation I would prefer to not do a large try/catch block. I would prefer to leave this bit alone, since logically it will always be assigned before being used. I agree, there's probably a better way to do this, but it would require some restructuring of the logic, which I'm not ireally interested in doing atm.

Or, you could probably set `projectionInfo = None` outside the `if` statement, and put the debug log back in the if block. That assigns the variables and prevents pyCharm from yelling whatever obscenities it yells with this kind of 'error'. I would be okay with that, and it looks cleaner. :)

Otherwise, everything else looks good, and I plan to merge this once this is solved.